README.md

```diff
@@ -30,7 +38,7 @@ app_keys = {
 
 access_keys = {
   'jonathankoren' : {	    # a specific account. You can change this, but
```

Comment:
You know, this is really a hack. I don't really like my login hard coded here. Perhaps it should be specified in secrets.py or some other config. The reason why it's like this is because it was originally code just for me. But now we should think about changing it.